plugins/modules/zos_job_query.py

```diff
@@ -272,7 +287,12 @@
             "creation_date": "2023-05-03",
             "creation_time": "12:13:00",
             "queue_position": 3,
-            "execution_time": "00:00:02"
+            "execution_time": "00:00:02",
```

Comment:
Thanks for adding an example `execution_time` even when it wasn't originally part of the scope for this PR.